lib/warder/style_guide_runner.rb

```diff
@@ -3,7 +3,7 @@ module Warder
   class StyleGuideRunner < Runner
     CLI_OPTION = 'g'
     CLI_FULL_OPTION = 'style-guide'
-    DESCRIPTION = 'Run style guide validation'
+    DESCRIPTION = 'Run ruby style guide validation'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.